src/SprykerEco/Zed/Computop/Business/ComputopBusinessFactory.php

```diff
@@ -511,6 +546,14 @@ public function createPostSavePayPal(): InitMapperInterface
         return new InitPayPalMapper($this->getConfig(), $this->getComputopApiService());
     }
 
+    /**
+     * @return \SprykerEco\Zed\Computop\Business\Hook\Mapper\Init\InitMapperInterface
+     */
+    public function createPostSavePayPalExpress(): InitMapperInterface
```

Comment:
Maybe will be better `createPostSavePayPalExpress` -> `createPostSavePayPalExpressMapper`